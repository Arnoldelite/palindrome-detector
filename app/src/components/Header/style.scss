.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
  height: 70px;
  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.11);
  &__left {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    .logo {
      width: 150px;
      padding-left: 40px;
    }
  }
  &__right {
    margin: 20px;
  }
}
.darken {
  color: black;
}
