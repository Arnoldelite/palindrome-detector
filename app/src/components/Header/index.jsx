import React from 'react'
import { Link } from 'react-router-dom'
import logo from '../../assets/batman-new.png'
import './style.scss'

const Header = () => (
  <div className="header">
    <div className="header__left">
      <Link to="/popular">
        <img className="logo" src={logo} alt="" />
        <sup className="darken">wayne ent.</sup>
      </Link>
    </div>
    <div className="header__right">
      <Link to="/app">
        <h3 className="darken">Palindrome Detector</h3>
      </Link>
    </div>
  </div>
)

export default Header
