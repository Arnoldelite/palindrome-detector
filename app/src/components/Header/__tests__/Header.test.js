import React from 'react'

// import Parse from 'parse/node'
import { shallow } from 'enzyme'

import Header from '..'
// import { expect } from 'chai';
// import sinon from 'sinon';

// import MyComponent from './MyComponent'
// import Foo from './Foo'

describe('<MyComponent />', () => {
  test('renders <Header /> component', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })

})
