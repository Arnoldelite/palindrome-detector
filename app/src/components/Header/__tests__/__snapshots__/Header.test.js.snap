// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders <Header /> component 1`] = `
<div
  className="header"
>
  <div
    className="header__left"
  >
    <Link
      replace={false}
      to="/popular"
    >
      <img
        alt=""
        className="logo"
        src="batman-new.png"
      />
      <sup
        className="darken"
      >
        wayne ent.
      </sup>
    </Link>
  </div>
  <div
    className="header__right"
  >
    <Link
      replace={false}
      to="/app"
    >
      <h3
        className="darken"
      >
        Palindrome Detector
      </h3>
    </Link>
  </div>
</div>
`;
